### PR TITLE
fix(ci): PaddlePaddle Intergration test

### PR DESCRIPTION
**Pull request by aarnphm**
## Checklist:

- [x] My code follows the bentoml code style, both `./dev/format.sh` and
  `./dev/lint.sh` script have passed
  ([instructions](https://github.com/bentoml/BentoML/blob/master/DEVELOPMENT.md#style-check-and-auto-formatting-your-code)).
- [x] My change reduces project test coverage and requires unit tests to be added
- [x] I have added unit tests covering my code change
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: tests/integration/projects/paddle/service.py [hunk 1]
```diff
@@ -33,10 +33,8 @@ def predict(self, df):
 if __name__ == "__main__":
     artifacts_path = sys.argv[1]
     bento_dist_path = sys.argv[2]
-    service = PaddleService()
-
-    from model.model import Model  # noqa # pylint: disable=unused-import
```
Comment (bojiang) [inline tests/integration/projects/paddle/service.py:38]: Do you know the purpose for this line?

**Review by bojiang — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -157,7 +157,7 @@ jobs:
       - name: Install test dependencies
         run: ./ci/install_test_deps.sh.cmd
       - name: Install paddle
-        run: python -m pip install paddlepaddle
+        run: python -m pip install paddlepaddle gast==0.3.3
```
Comment (bojiang) [inline .github/workflows/ci.yml:160]: Why gast here?

File: bentoml/frameworks/paddle.py [hunk 3]
```diff
@@ -27,6 +26,7 @@ class PaddlePaddleModelArtifact(BentoServiceArtifact):
     Example usage:
 
     >>> import pandas as pd
+    >>> import numpy as np
```
Comment (bojiang) [inline bentoml/frameworks/paddle.py:29]: [thumbup]

File: DEVELOPMENT.md [hunk 4]
```diff
@@ -241,6 +241,9 @@ cd {PROJECT_ROOT}/bentoml/yatai/web/
 npm run build
 ```
 
+## How to test GitHub Actions locally
+
+If you are developing new artifacts or modify GitHub Actions CI (adding integration test, unit tests, etc), use [`nektos/act`](https://github.com/nektos/act) to run Actions locally.
```
Comment (yubozhao) [inline DEVELOPMENT.md:246]: 👍🏼

File: .github/workflows/ci.yml (hunk 2 above)
Comment (aarnphm) [reply .github/workflows/ci.yml:160]: apparently when install `paddlepaddle` from pypi it install gast==0.5.5, which ran into errors in CI. `paddlepaddle` requires gast to be >=0.3.3 and <0.4.0, but I'm not entirely sure why it install `gast==0.5.5`, so thats why we install it there.

Comment (bojiang) [reply .github/workflows/ci.yml:160]: I see.

> but I'm not entirely sure why it install gast==0.5.5
Oh that's a known behavior of pip. BTW that's why we need dependency locking when building bento bundle.

**Review by bojiang — APPROVED** [review]

**Comment by bojiang** [comment]
Great we can get rid of the ❌ now!

**Pull request merged**